src/CreateSignalFactory.js

```diff
@@ -153,7 +153,24 @@ module.exports = function (controller, model, services, compute, modules) {
                   module: defaultOptions.modulePath.reduce(function (modules, key) {
                     return modules[key]
                   }, modulesArg)
-                })
+                }
+
+                if (utils.isDeveloping()) {
+                  try {
+                    actionFunc(actionArg)
+                  } catch (e) {
+                    action.error = {
+                      name: e.name,
+                      message: e.message,
+                      stack: actionFunc.toString()
+                    }
+                    controller.emit('signalError', {action: action, signal: signal})
```

Comment:
Because it is the signal that is aborted, not only the action. The signal is aborted because of an action, which is also passed on the event :)